website/content/partials/telemetry-metrics/vault/audit/log_response_failure.mdx

```diff
@@ -2,11 +2,11 @@
 
 | Metric type | Value  | Description                                             |
 |-------------|--------|---------------------------------------------------------|
-| counter     | number | Number of audit log request failures across all devices |
+| counter     | number | Number of audit log response failures across all devices |
 
 The number of request failures is a **crucial metric**.
 
-A non-zero value for `vault.audit.log_response_failure` indicates that one of
+A non-zero value for `vault.audit.log_response_failure` indicates that all of
 the configured audit log devices failed to respond to Vault. If Vault cannot
```

Comment:
```suggestion
the configured audit log devices failed to log a response to a request to Vault. If Vault cannot
```
Hey Gerard, I was having a look at the PR and I noticed this next line is also incorrect, so here is a quick suggestion to fix that up. Thanks for making our docs better, btw!